examples/with_bevy/src/main.rs

```diff
@@ -46,16 +46,19 @@ fn render_scenes(
 ) {
     for scene in &mut scenes {
         let gpu_image = gpu_images.get(&scene.1).unwrap();
-
+        let params = vello::RenderParams {
+            clear_color: vello::peniko::Color::AQUAMARINE,
```

Comment:
Is it a clear colour or a base colour? For me, clear colour suggests an extra step in the pipeline, running a clearing step.

It's probably a distinction without a difference

Reply:
I certainly had the same question as I settled on clear color, as it is slightly misleading if you think of it as the clear blit in a render pass.  Base color sounds good to me.